### DISCUSSION TITLE
Raða eftir:

### DISCUSSION THREAD

**Discussion by binnih** [hugihlynsson/veldu-rafbil#443] [Ideas]
Mæli eindregið með að bæta við "acending/decending" möguleika þegar notandi listar eftir "Nafni Verði Drægni Hröðun Verði á km"

**Comment by hugihlynsson** [comment]
Þetta er mjög góð hugmynd! Ég sleppti því upphaflega vegna þess að það voru svo fáir bílar og það var einfalt að skrolla bara neðst og svo upp ef maður vildi fá öfuga röð en það er erfiðara núna þegar úrvalið er orðið svona gott. Skal græja þetta við tækifæri!

**Reply by binnih** [reply]
Frábært, takk!
